setup.py

```diff
@@ -4,7 +4,25 @@
     from setuptools import setup
 except ImportError as ex:
     from distutils.core import setup
-from odml.info import AUTHOR, CONTACT, CLASSIFIERS, HOMEPAGE, VERSION
+
+try:
+    from odml.info import AUTHOR, CONTACT, CLASSIFIERS, HOMEPAGE, VERSION
```

Comment:
Now that I see it, perhaps it's better to just do the `except` block and not try to import.

I wonder if it's possible to end up in a situation where the setup imports an exiting installation that has a different version number.

Since the parsing code is there, might as well, right?